src/components/DeleteDoctor.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const DeleteDoctor = ({ doctor }) => {
+  function handleDelete() {
```

Comment:
We should use ES6 syntax for all the functions:

`const handleDelete = () => {...}` 

_**NOTE:**_ When you are implementing this function, make sure the doctor is deleted from the redux store as well as the backend database.